internal/provider/query_specification_data_source_test.go

```diff
@@ -210,6 +211,7 @@ data "honeycombio_query_specification" "test" {
     column = "we-should-not-specify-a-column-with-COUNT"
   }
 }`,
+		PlanOnly:    true,
```

Comment:
All of these `PlanOnly: true` additions are just to speed things up here a bit: no external resources need to be created.